src/readpst.c

```diff
@@ -320,7 +323,8 @@ void process(pst_item *outeritem, pst_desc_tree *d_ptr)
                 if (mode == MODE_SEPARATE) close_separate_file(&ff);
             }
 
-        } else if (item->email && ((item->type == PST_TYPE_NOTE) || (item->type == PST_TYPE_SCHEDULE) || (item->type == PST_TYPE_REPORT))) {
+        } else if (item->email && ((item->type == PST_TYPE_NOTE) || (item->type == PST_TYPE_SCHEDULE) || (item->type == PST_TYPE_REPORT) || item->type == PST_TYPE_STICKYNOTE || item->type == PST_TYPE_TASK)) {
```

Comment:
It looks like these can be handled as emails and otherwise they are just omitted.